pep-0001.txt

```diff
@@ -436,14 +426,29 @@ Each PEP should have the following parts:
    The rationale should provide evidence of consensus within the
    community and discuss important objections or concerns raised
    during discussion.
+   
+5. Specification -- The technical specification should describe the
+   syntax and semantics of any new language feature.  The
+   specification should be detailed enough to allow competing,
+   interoperable implementations for at least the current major Python
+   platforms (CPython, Jython, IronPython, PyPy).
 
-7. Backwards Compatibility -- All PEPs that introduce backwards
+6. Backwards Compatibility -- All PEPs that introduce backwards
    incompatibilities must include a section describing these
    incompatibilities and their severity.  The PEP must explain how the
    author proposes to deal with these incompatibilities.  PEP
    submissions without a sufficient backwards compatibility treatise
    may be rejected outright.
 
+7. How to Teach This -- For a PEP that adds new functionality or changes
```

Comment:
Yes!!!

Reply:
That was actually already there, but people have not been doing it. 😉